geoh5py/workspace/workspace.py

```diff
@@ -827,6 +827,13 @@ def objects(self) -> list[objects.ObjectBase]:
         return self._all_objects()
 
     def open(self, mode: str | None = None):
+        if isinstance(self._geoh5, h5py.File) and self._geoh5:
```

Comment:
**suggestion:** no need to test `self._geoh5`
if `isinstance(self._geoh5, h5py.File)` is true, then `self._geoh5` is not None.

Reply:
Indeed!